subprojects/scala/src/integTest/groovy/org/gradle/scala/scaladoc/ScalaDocIntegrationTest.groovy

```diff
@@ -71,4 +74,32 @@ class ScalaDocIntegrationTest extends AbstractIntegrationSpec implements Directo
         skipped scaladoc
     }
 
+
+    @ToBeFixedForInstantExecution
+    def "scaladoc uses maxMemory"() {
+        buildFile << """
+            apply plugin: "scala"
+            scaladoc.maxMemory = '234M'
+
+            repositories {
+                mavenCentral()
+            }
+
+            dependencies {
+                implementation 'org.scala-lang:scala-library:2.11.12'
+            }
+        """
+
+        writeSourceFile()
+        when:
+        ExecutionResult result = run scaladoc, "-d"
```

Comment:
The information on the daemon is not showed with `-i`.

Wondering if that should be changed in the future.

Also, I found that for tasks that use the worker api such as this one do not mention that the default is `512m` wonder if this should be surfaced when the task is executed, adding the worker info level logging would definitely help